website/client/src/components/challenges/findChallenges.vue

```diff
@@ -171,7 +171,23 @@ export default {
   computed: {
     ...mapState({ user: 'user.data' }),
     filteredChallenges () {
-      return this.challenges;
+      const { filters } = this;
```

Comment:
The code is identical between the two Challenge Vue files, now--could this be abstracted as a mixin?

Reply:
We already have the `challengeUtilities` mixin, so it could live there